src/platform_impl/windows/dark_mode.rs

```diff
@@ -169,27 +169,20 @@ fn should_apps_use_dark_mode() -> bool {
 }
 
 fn is_high_contrast() -> bool {
-    let mut hc = winuser::HIGHCONTRASTA {
+    let mut hc = HIGHCONTRASTA {
         cbSize: 0,
         dwFlags: 0,
         lpszDefaultScheme: std::ptr::null_mut(),
     };
 
     let ok = unsafe {
-        winuser::SystemParametersInfoA(
-            winuser::SPI_GETHIGHCONTRAST,
+        SystemParametersInfoA(
+            SPI_GETHIGHCONTRAST,
             std::mem::size_of_val(&hc) as _,
             &mut hc as *mut _ as _,
             0,
         )
     };
 
-    ok != FALSE && (winuser::HCF_HIGHCONTRASTON & hc.dwFlags) == 1
-}
-
-fn widestring(src: &'static str) -> Vec<u16> {
-    OsStr::new(src)
-        .encode_wide()
-        .chain(Some(0).into_iter())
-        .collect()
+    ok == true.into() && util::has_flag(hc.dwFlags, HCF_HIGHCONTRASTON)
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)